wondrous-app/components/CreateEntity/CreateEntityModal/TaskTemplatePicker/styles.tsx

```diff
@@ -0,0 +1,519 @@
+  }
+`;
+
+export const TaskTemplateEllipsesIcon = styled(EllipsesIcon)`
+  width: 24px;
+  color: #ffffff;
+  background: rgba(122, 122, 122, 0.2);
+  height: 24px;
+  padding: 6px;
+  border-radius: 6px;
+  path {
+    background: rgba(122, 122, 122, 0.2);
+  }
+  :hover {
+    cursor: pointer;
+  }
+`;
+
+export const TaskTemplateContainer = styled.div`
+  border-radius: 4px;
+  background: #121212;
+  padding: 14px;
+  margin-bottom: 12px;
+  :hover {
+    background: rgba(122, 122, 122, 0.2);
+  }
+`;
+export const TaskTemplateBorderLine = styled.div`
+  border-bottom: 0.5px dashed #4b4b4b;
+  margin-bottom: 12px;
+`;
+export const PodSearchDefaultImage = styled((props) => (
+  <div {...props}>
+    <PodIcon />
+  </div>
+))`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: ${({ color }) => color};
+  border-radius: 50px;
+  width: 20px;
+  height: 20px;
+  svg {
+    width: 20px;
+    height: 20px;
+  }
+`;
+
+export const TaskTemplateLabel = styled(Typography)`
+  && {
+    font-family: 'Space Grotesk';
+    font-size: 13px;
+    font-weight: 500;
+    color: ${palette.white};
+    margin-left: 6px;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+  }
+`;
+
+export const TaskTemplateRewardContainer = styled.div`
+  && {
+    flex-direction: row;
+    display: flex;
+  }
+`;
+
+export const TaskTemplateLabelValue = styled((props) => <Typography {...props} />)`
+  && {
+    font-family: 'Space Grotesk';
+    font-size: 13px;
+    font-weight: 100;
+    color: #ffffff;
+    margin-bottom: 4px;
+    overflow: hidden;
+  }
+`;
+
+export const TaskTemplateRewardValue = styled(Typography)`
+  && {
+    font-family: 'Space Grotesk';
+
+    font-size: 13px;
+    font-weight: 500;
+    ${({ theme }) => `
+      color: ${theme.palette.green800};
+    `}
+  }
+`;
+
+export const TaskTemplatePointsValue = styled(Typography)`
+  && {
+    font-family: 'Space Grotesk';
+
+    font-size: 13px;
+    font-weight: 500;
+    background: linear-gradient(180deg, #ffffff 0%, #ffd653 100%);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+  }
+`;
+
+export const TaskTemplateRewardBox = styled(Typography)`
+  && {
+    flex-direction: row;
+    display: flex;
+    align-items: center;
+    padding: 8px;
+    border-radius: 6px;
+    background: rgba(122, 122, 122, 0.2);
+    margin-bottom: 4px;
+    margin-top: 8px;
+    margin-right: 4px;
+  }
+`;
+
+export const CreateEntityDefaultDaoImage = styled(PointsIconCustom)`
+  color: linear-gradient(180deg, #ffffff 0%, #ffd653 100%);
+  background: #000000;
+  width: 24px;
+  height: 24px;
+  padding: 6px;
+  border-radius: 20px;
+  margin-right: 8px;
+`;
+
+export const TaskTemplateEthereumIcon = styled(Ethereum)`
+  background: #000000;
+  width: 24px;
+  height: 24px;
+  padding: 6px;
+  border-radius: 20px;
+  margin-right: 8px;
+`;
+
+export const TaskTemplateOptionsLabel = styled(Typography)`
```

Comment:
also one more since this component has onClick event can we set this as a button ?